canvas_modules/common-canvas/src/toolbar/toolbar.scss

```diff
@@ -35,6 +35,70 @@ $toolbar-divider-width: 1px;
 
 	/* Prevent elements from being dragged onto canvas */
 	-webkit-user-drag: none;
+	&.toolbar-size-small {
+		.toolbar-left-bar {
+			align-items: center;
+			text-align: center;
+			padding: 0px;
+		}
+		.toolbar-item {
+			&.default,
+			&.ghost {
+				& button {
+					padding: 3px;
+				}
+			}
+		}
+		.toolbar-overflow-item {
+			background-color: $ui-01;
+			display: inline-block;
+			& button {
+				padding:0;
+				height: $toolbar-button-height;
+				min-height: 30px;
+				background-color: $ui-01;
+				border-right: $toolbar-divider-width solid $ui-03;
+			}
+			& button:hover {
+				background-color: $hover-field;
+			}
+			& button:disabled:hover {
+				background-color: $ui-01;
+			}
+		}
+		.toolbar-overflow-menu-item button {
+			width: 100%;
+			padding-top: 2px;
+			padding-bottom: 2px;
+			min-height: 20px;
+		}
+		.toolbar-item-content {
+			width: inherit;
+			display: flex;
+			justify-content: center;
+			padding: 5px;
+			&.overflow {
+				justify-content: left;
+				padding: 8px 0;
+			}
+			&.default {
+				color: $icon-01; // Used by label and pick up as currentColor in icons.
+			}
+			&.disabled.default {
+				color: $disabled-02;
+				fill: $disabled-02; // For custom svg images
+			}
+			.toolbar-icon {
+				height: $toolbar-icon-size;
+				& svg {
+					height: $toolbar-icon-size;
+					width: $toolbar-icon-size;
+					/* Prevent elements from being dragged onto canvas */
+					-webkit-user-drag: none;
+				}
+			}
+		}
+	}
 }
```

Comment:
Regarding Carita's comments above. I think what you ought to do is take the SCSS that is in table-buttons.scss as a starting point for this SCSS.  You can insert that SCSS inside this section: 
```
&.toolbar-size-small {
}
```
This will isolate those styles from the SCSS for medium size buttons. Then do some testing to see if things are working when the "small" size toolbar is displayed in common-properties.